### PR TITLE
Improve ambiguous "more info" label (fixes #1074)

### LINKED ISSUE #1074: Accessibility issue: ambiguous link text in settings modal
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
more info link inside settings modal

### Issue description
There is a link at the bottom of the settings modal to the UV github page. The visible label is "more info" and this is what a screenreader will read out. It is impossible to tell what this more info is and where the link leads: neither link text nor context give a hint. Even a user actually looking for a link to UV or their repo may miss this link.

This is actually an issue for ALL users but users who are not navigating by mouse much more so, because mouse user can hover over the mysterious link and see the web address displayed (in most browsers). 
A screenreader or keyboard user will have to use complicated workarounds to find out where this link leads (proably easiest to just click on it). 

### Steps to reproduce
1. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
2. Activate to the settings modal
3. Find the link to the github repo

### Expected behaviour
Link text should be more descriptive, for example "more info about Universal Viewer". 

### Possible fix
Change the visible link text. 
Note that just adding an accessible label without changing the visible text would not fix this issue for everyone, and may even introduce other problems (accessible labels must contain the visible label, see WCAG Criterion 2.5.3 Label in Name) 

### WCAG criterion
2.4.4 Link Purpose (In Context) (Level A)

### Related code

```html
<a href="https://github.com/universalviewer/universalviewer" target="_blank">more info</a>
```

### Other information
**Note**: Although there is an exception in the rule stating this behaviour is acceptable in instances where the link text is ambiguous to all users (not just disabled), I would not apply the exception here because the link is surely not meant to be ambiguous and also because the workaround for sighted users is so much easier and quicker than screenreader users. 
And most of all: Fixing the link text will make UV better for everyone.

### PR DESCRIPTION
**Pull request by Geoffsc**
Fix for #1074

### CODE REVIEW

**Comment by Geoffsc** [comment]
@jamesmisson @demiankatz

**Comment by demiankatz** [comment]
Thanks, @Geoffsc, this makes sense to me -- but we'll need to update the other locales as well. Can @LlGC-szw help with that?

**Comment by LlGC-szw** [comment]
Happy to assist.

File: src/locales/en-GB.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
   "$clickToZoomEnabled": "Mouse Click To Zoom",
   "$reducedMotion": "Reduce motion (disables animations)",
   "$preserveViewport": "Preserve Zoom",
-  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer'>more info</a>",
+  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer'>more info about the Universal Viewer</a>",
```
Comment (jamesmisson) [inline src/locales/en-GB.json:118]: I suggest we capitalise the first letter to bring the formatting inline with other UV text ('More...')

**Comment by demiankatz** [comment]
> Happy to assist.

Thanks! What's the best way to do this? Do you want to just add the translations in a comment here so that @Geoffsc can insert them into the project, or do you want to push updates directly to the branch here?

File: src/locales/en-GB.json (hunk 1 above)
Comment (demiankatz) [reply src/locales/en-GB.json:118]: No objection here!

Comment (Geoffsc) [reply src/locales/en-GB.json:118]: Agreed! Happy to adjust along with updating the translations

**Review by demiankatz — APPROVED** [review]
Thanks, everyone, looks like this is good to go!

**Review by demiankatz — CHANGES_REQUESTED** [review]
Well, except that something weird has happened and there are conflicts listed in all the language files...

@Geoffsc, let me know if you need help straightening this out. I wonder if it's something to do with line breaks or whitespace.

**Comment by Geoffsc** [comment]
> Well, except that something weird has happened and there are conflicts listed in all the language files...
> 
> @Geoffsc, let me know if you need help straightening this out. I wonder if it's something to do with line breaks or whitespace.

I updated dev locally and merged it into this branch. It looks like the language files may have been updated in dev recently to address another issue. Do we know which one? In any case I think I can resolve by accepting the upstream change and then applying my updates since they're just one line per file.

image

**Comment by Geoffsc** [comment]
@demiankatz this should be good to go now.

**Review by demiankatz — APPROVED** [review]
Looks good; thanks for straightening that out, @Geoffsc!

**Pull request merged**